pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -78,6 +81,8 @@ func NewCmdAddRoleToUser(name, fullName string, f *clientcmd.Factory, out io.Wri
 	}
 
 	cmd.Flags().StringVar(&options.RoleNamespace, "role-namespace", "", "namespace where the role is located: empty means a role defined in cluster policy")
+	cmd.Flags().Var(&saNamesLong, "service-account", "service account in the current namespace to use as a user")
```

Comment:
@smarterclayton is there a cleaner way to have aliases for flags?  I'd really like `--service-account` and `--sa`